scripts/zones/Aht_Urhgan_Whitegate/npcs/Kaduru-Haiduru.lua

```diff
@@ -0,0 +1,125 @@
+-----------------------------------
+-- Area: Aht Urhgan Whitegate
+-- NPC: Kaduru-Haiduru
+-- Teleport NPC
+-----------------------------------
+
+-----------------------------------
+-- onTrigger
```

Comment:
please no header block comments that contain only function names, we got rid of those